test/test_linalg.py

```diff
@@ -33,6 +33,18 @@
     import scipy
 
 class TestLinalg(TestCase):
+    def setUp(self):
+        super(self.__class__, self).setUp()
+        torch.backends.cuda.matmul.allow_tf32 = False
+        self.precision_overrides = {
+            torch.float: 1e-4,
```

Comment:
Does it mean that regular fp32 needs expanded tolerance? How are tests passing currently?

Reply:
Ohh, yes you're correct. I forgot to delete them. Let me modify this.